### YamlMime:PythonClass
uid: azure.messaging.webpubsubclient.models.OpenClientError
name: OpenClientError
fullName: azure.messaging.webpubsubclient.models.OpenClientError
module: azure.messaging.webpubsubclient.models
inheritances:
- azure.core.exceptions.AzureError
summary: Exception raised when fail to start the client
constructor:
  syntax: 'OpenClientError(message: object | None, *args: Any, **kwargs: Any)'
  parameters:
  - name: message
    isRequired: true
methods:
- uid: azure.messaging.webpubsubclient.models.OpenClientError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.messaging.webpubsubclient.models.OpenClientError.raise_with_traceback
  name: raise_with_traceback
  summary: 'Raise the exception with the existing traceback.


    *Deprecated since version 1.22.0:* This method is deprecated as we don''t support
    Python 2 anymore. Use raise/from instead.'
  signature: raise_with_traceback() -> None
- uid: azure.messaging.webpubsubclient.models.OpenClientError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.messaging.webpubsubclient.models.OpenClientError.args
  name: args
